### ISSUE TITLE
Add @Abrarsyeds SpongeSDK

### ISSUE THREAD

**Issue by ryantheleach** [SpongePowered/SpongeDocs#209]
@AbrarSyed added a SpongeSDK into the sponge forge mod project that uses ForgeGradle to set up a workspace for a project.

This may be easier then getting people to set it up manually for plugin development.

According to gab, it's probably blocked by this PR : https://github.com/MinecraftForge/FML/pull/653

**Comment by gabizou** [comment]
At the current moment, the SDK will NOT work due to Sponge not releasing what's called "dev" builds of Sponge where we do NOT re-obfuscate our mod with the mcp mappings. If we did, we'd be able to use the SDK, and I would have provided a video using the SDK. Until that PR is merged, I can not provide any documentation with regards to using the SDK for both testing sponge and plugin purposes.

**Comment by AbrarSyed** [comment]
Dev time deobf is in ForgeGradle as of FG2, and that fnl PR is no longer relevant.

https://github.com/MinecraftForge/MinecraftForge/blob/master/mdk/build.gradle#L53

**Comment by ryantheleach** [comment]
@gabizou What needs to happen for the SDK to work now?

**Comment by simon816** [comment]
I have updated the SDK to use SpongeAPI 3.0.0.
https://github.com/SpongePowered/SpongeForge/compare/update/sdk

Unfortunately there's a problem with there being a dependency on spongecommon (which is invalid) so waiting on @Minecrell to merge this branch
https://github.com/SpongePowered/SpongeCommon/compare/feature/deobf-jar

**Comment by Tzky** [comment]
What's the status of this issue?

**Comment by Minecrell** [comment]
Closing for the reasons explained in https://github.com/SpongePowered/SpongeForge/issues/377#issuecomment-193343570, this will be replaced by something in SpongeGradle which will then be added to the SpongeGradle docs introduced with #456
